docker/bin/zookeeperStart.sh

```diff
@@ -20,6 +20,10 @@ MYID_FILE=$DATA_DIR/myid
 LOG4J_CONF=/conf/log4j-quiet.properties
 DYNCONFIG=$DATA_DIR/zoo.cfg.dynamic
 
+# Copy over the zoo.cfg file in the configmap to make it writable
+ZOOCFGDIR=$DATA_DIR
```

Comment:
Is it safe to set this environment variable here or could it be overwritten in some situations? Moreover, doing a `printenv` in a Zookeeper pod I see: `ZOO_CONF_DIR=/conf` and `ZOOCFGDIR=/conf`. Do we need to overwrite both?

Reply:
I think this needs to be changed in zkEnv.sh.